cmd/restic/cmd_backup.go

```diff
@@ -81,6 +82,7 @@ func init() {
 	f.StringArrayVar(&backupOptions.Tags, "tag", nil, "add a `tag` for the new snapshot (can be specified multiple times)")
 	f.StringVar(&backupOptions.Hostname, "hostname", "", "set the `hostname` for the snapshot manually")
 	f.StringVar(&backupOptions.FilesFrom, "files-from", "", "read the files to backup from file (can be combined with file args)")
+	f.StringVar(&backupOptions.TimeStamp, "time", "", "time of the backup (ex. 2012-11-01T22:08:41+00:00) (default: now)")
```

Comment:
I'm not sure we should use this rather complicated format. What about using the time format that is used in the `snapshots` command? It would require proper quoting (since there's a space in it) but is much easier to get right.

https://github.com/restic/restic/blob/a3f6bf3e5a07949e0a30f1fd05c6d75affe51c08/cmd/restic/table.go#L46